src/cli/commands/install.js

```diff
@@ -934,16 +940,10 @@ export async function install(config: Config, reporter: Reporter, flags: Object,
 }
 
 export async function run(config: Config, reporter: Reporter, flags: Object, args: Array<string>): Promise<void> {
-  let lockfile;
-  let error = 'installCommandRenamed';
-  if (flags.lockfile === false) {
-    lockfile = new Lockfile();
-  } else {
-    lockfile = await Lockfile.fromDirectory(config.lockfileFolder, reporter);
-  }
-
   if (args.length) {
-    const exampleArgs = args.slice();
+    let command = 'add';
+    let error = 'installCommandRenamed';
```

Comment:
Is this change relevant to the new feature?

Reply:
No. Guess I'll remove the refactor.